source/integrate/community-for-mattermost-for-microsoft-teams.rst

```diff
@@ -0,0 +1,71 @@
+- **Stay Updated**: Keep up with the latest Mattermost announcements, updates, and roadmap information
+
+This app is designed to work with Microsoft 365, Outlook and Microsoft Teams.
+
+Setup
+-----
+
+Install the Community for Mattermost for Microsoft 365 application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
+
+Any Microsoft 365 user can complete the steps below to access the Community for Mattermost within their Microsoft environment.
+
+1. Sign in to your Microsoft Teams account from a `browser <https://teams.microsoft.com/v2/?clientexperience=t2>`_ or the desktop application.
+
+2. Select the **[+] Apps** button in the Teams sidebar.
+
+3. Search for "Community for Mattermost" and then select **Add** to install the application.
+
+4. (Optional) Pin the application to your Teams sidebar by right-clicking on it and selecting **Pin**.
+
+5. Once the Community for Mattermost app is installed, you'll be connected to the public Mattermost Community instance.
+
+Getting Started with the Community for Mattermost
+-------------------------------------------------
+
+Once you've installed the app, here are some ways to get started:
+
+1. **Ask questions**: Use the `~ask-anything <https://community.mattermost.com/core/channels/ask-anything>`_ channel to ask technical questions or get support from the community.
+
+2. **Stay informed**: Follow the `~release-announcements <https://community.mattermost.com/core/channels/release-announcements>`_ channel to keep up with the latest news about upcoming releases and updates from the Mattermost team.
+
+3. **Contribute**: If you're interested in contributing to Mattermost, join the `~developers <https://community.mattermost.com/core/channels/developers>`_ channel to connect with other contributors.
+
+4. **Share feedback**: Your ideas and feedback are valuable! Share your thoughts in the `~user-feedback <https://community.mattermost.com/core/channels/user-feedback>`_ channel.
+
+FAQ
+---
+
+Where can I get support for the Community for Mattermost for Microsoft 365 app?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can browse existing open issues or submit a new issue for support `here <https://github.com/mattermost/mattermost-teams-tab/issues>`_.
+
+Do I need a Mattermost account to use this application?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Yes, you'll need to create a free account on the Mattermost Community server when you first access it through the app.
+
+Can I use this app to connect to my organization's Mattermost server?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+No, this app is specifically designed to connect to the public Mattermost Community instance. If you're interested in connecting to your own Mattermost instance, please reach out to support@mattermost.com for assistance.
```

Comment:
Given that Support is a paid customer/admin only path, is that what we want to offer here?

Reply:
I do think this will be an enterprise-only feature. Is there a better "inbound" email address we could direct customers towards?

Reply:
Depends on intention: Do we imagine that someone wants to "talk to an expert" (aka Sales) or "get support" (aka Support)?